CPS-XXXX/README.md

```diff
@@ -0,0 +1,155 @@
+---
+CPS: ????
+Title: CPS-???? | Cardano URIs
+Status: Open
+Category: Tools
+Authors:
+  - Adam Dean <adam@crypto2099.io>
+Proposed Solutions:
+  - CIP-0013: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0013
+  - CIP-0099: https://github.com/cardano-foundation/CIPs/tree/master/CIP-0099
+  - CIP-0107: https://github.com/cardano-foundation/CIPs/pull/635
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/issues/836
+  - https://github.com/cardano-foundation/CIPs/pull/635
+  - https://github.com/cardano-foundation/CIPs/pull/841
```

Comment:
```suggestion
Discussions:
  - https://github.com/cardano-foundation/CIPs/issues/836
  - https://github.com/cardano-foundation/CIPs/pull/559
  - https://github.com/cardano-foundation/CIPs/pull/635
  - https://github.com/cardano-foundation/CIPs/pull/841
```
The added thread #559 is the last time the scope & limitations of the URI syntax were discussed with the intent of finalising them... which is consistent with what you've inventoried in the _Registered URI Authorities_ section.

Reply:
Looks good! Let's add it!